python/chronos/src/bigdl/chronos/data/utils/public_dataset.py

```diff
@@ -268,14 +275,21 @@ def download(url, path, chunk_size):
     """
     from bigdl.nano.utils.log4Error import invalidInputError
     req = requests.get(url, stream=True)
-    file_size = int(req.headers['content-length'])
+    file_size = bytes_convert(int(req.headers['content-length']))
     invalidInputError(req.status_code == 200, "download failure, please check the network.")
     file_name = url.split('/')[-1]
-    pbar = tqdm.tqdm(total=file_size, unit='B', unit_scale=True, desc=file_name)
+    logger.info(f"Start download {file_name.partition('.')[0]}, file size: {file_size}")
     with open(os.path.join(path, file_name), 'wb') as f:
         for chunk in req.iter_content(chunk_size):
             if chunk:
                 f.write(chunk)
-                pbar.update(chunk_size)
                 f.flush()
-    pbar.close()
+
+
+def bytes_convert(size: int):
+    if size / 1024 < 1024:
+        return f"{(size / 1024):<.2f}K"
+    if size / 1024 ** 2 < 1024:
+        return f"{(size / 1024**2):<.2f}M"
+    else:
+        return f"{(size / 1024**3):<.2f}G"
```

Comment:
how does it look when user call this API?

Reply:
Sorry, I forgot to add the example.
```python
In [3]: from bigdl.chronos.data.repo_dataset import get_public_dataset

In [4]: df = get_public_dataset('nyc_taxi', with_split=True, redownload=True)
2022-09-26 14:37:24,342 - Start download nyc_taxi, file size: 52.40K

In [6]: df = get_public_dataset('fsi', with_split=True, redownload=True)
2022-09-26 14:47:59,482 - Start download individual_stocks_5yr, file size: 9.67M

In [7]: df = get_public_dataset('AIOps', with_split=True, redownload=True)
2022-09-26 14:48:21,838 - Start download machine_usage, file size: 1.65G

In [9]: df = get_public_dataset('uci_electricity', with_split=True, redownload=True)
2022-09-26 14:48:59,895 - Start download LD2011_2014, file size: 249.23M
```

This API recalculates the size of the file and displays the appropriate units, the smallest file is nyc_taxi, 52.40K.